### PR TITLE
Xkbcomp: update several

### PR DESCRIPTION
**Pull request by dkwo**
Trying to address issue #31870
not completely solved, yet

### CODE REVIEW

File: srcpkgs/xorgproto/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'xorgproto'
 pkgname=xorgproto
-version=2020.1
+version=2021.4.99.2
```
Comment (ericonr) [inline srcpkgs/xorgproto/template:3]: I tend to read "99" as beta releases. Is that not the case?

**Review by ericonr — COMMENTED** [review]
I think the commit messages for the revbumps should mention their intentions, maybe?

@leahneukirchen ok with the PR?

File: srcpkgs/xorgproto/template (hunk 1 above)
Comment (leahneukirchen) [reply srcpkgs/xorgproto/template:3]: Yes, I'd suggest to use 2021.4 instead.

Comment (dkwo) [reply srcpkgs/xorgproto/template]: Done.

**Comment by dkwo** [comment]
Maybe just wait a bit to merge?
I think warnings should go away if rebuild `libX11`.

**Comment by ericonr** [comment]
Ok, let me know when you have it in order :)

**Review by leahneukirchen — APPROVED** [review]

File: srcpkgs/xkbcomp/template [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 pkgname=xkbcomp
-version=1.4.2
+version=1.4.5
 revision=1
 build_style=gnu-configure
 hostmakedepends="pkg-config bison"
 makedepends="libX11-devel libxkbfile-devel"
 short_desc="XKBD keymap compiler"
-homepage="https://www.x.org/"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (ericonr) [inline srcpkgs/xkbcomp/template:9]: Wild :o

**Comment by ericonr** [comment]
@dkwo The builders should respect the build order for this...

**Comment by ericonr** [comment]
And I might have jumped the gun, sorry if I did.

**Comment by dkwo** [comment]
No worries.

**Pull request merged**